trustar_connector.py

```diff
@@ -195,6 +194,10 @@ def _make_rest_call_helper(self, endpoint, action_result, headers={}, params=Non
         """
 
         retry_failure_flag = False
+
+        if headers is None:
+            headers = {}
```

Comment:
What do you think about:
```
if not headers:
  headers = {}
```
here we can also turn other False value into an empty object

Reply:
Done